bootstraptool/pom.xml

```diff
@@ -138,7 +138,50 @@
                 </plugins>
             </build>
         </profile>
-
+        <profile>
```

Comment:
I think gobi/pre-gobi profiles are not useful considering that those sidechains are already registered and we're unlickely going to use bootstraptool for them.